includes/class-ceske-sluzby-zasilkovna.php

```diff
@@ -0,0 +1,106 @@
+<?php
+// http://docs.woothemes.com/document/settings-api/
+// http://speakinginbytes.com/2014/07/woocommerce-settings-tab/
+class WC_Shipping_Ceske_Sluzby_Zasilkovna extends WC_Shipping_Method {
+
+  public function __construct() {
+     $this->id = 'ceske_sluzby_zasilkovna';
+     $this->method_title = 'Zásilkovna';
+     $this->method_description = 'Základní možnosti nastavení.';
+     $this->title = $this->get_option( 'zasilkovna_nazev' );
+     $this->enabled = $this->get_option( 'enabled' );
+     $this->init();
+   }
+ 
+  function init() {
+    $this->init_form_fields();
+    $this->init_settings();
+    add_action( 'woocommerce_update_options_shipping_' . $this->id, array( $this, 'process_admin_options' ) );
+  }
+ 
+  public function calculate_shipping( $package = array() ) {
+    $zeme = WC()->customer->get_shipping_country();
+    if ( $zeme == "CZ" ) { $cena = $this->get_option( 'zasilkovna_zakladni-cena' ); }
+    if ( $zeme == "SK" ) { $cena = $this->get_option( 'zasilkovna_zakladni-cena-slovensko' ); }
+    $rate = array(
+      'id' => $this->id,
+      'label' => $this->title,
+      'cost' => $cena
+    );
+    $this->add_rate( $rate );
+  }
+      
+  public function init_form_fields() {
+    $zakladni = array(
+      'enabled' => array(
+				'title'   => 'Povolit',
+				'type'    => 'checkbox',
+				'label'   => 'Aktivovat a zobrazit v nabídce dostupných možností dopravy.',
+				'default' => 'no'
+      ),
+      'zasilkovna_nazev' => array(
+				'title'       => 'Název',
+				'type'        => 'text',
+				'description' => 'Název pro zobrazení v eshopu.',
+				'default'     => 'Zásilkovna',
+				'css'         => 'width: 300px;'
+      ),
+      'zasilkovna_id-obchodu' => array(
```

Comment:
ID obchodu v kontextu zásilkovny znamená něco trochu jiného (textový řetězec, který identifikuje odesilatele při podávání přes API). Můžete to prosím upravit na API klíč? V zásilkovně to tak máme vedené všude a předejdeme zmatkům :)